libs/common/src/types/vault-timeout.type.ts

```diff
@@ -0,0 +1,17 @@
+// Note: the below comments are just for documenting what they used to be.
+export const VaultTimeoutStringType = {
```

Comment:
```suggestion
export const VaultTimeoutType = {
```
Maybe there's some distinct reason we're adding the explicitly state "String" data type here... but wouldn't it normally be extraneous information that is unnecessary to be concerned about?

Reply:
Hm, I think given the circumstances behind switching from one data type to another, it makes sense to be more explicit here. 

I guess the ideal would be to eventually refactor out the `number` data type, right? Regardless, I think keeping the current structure in place makes sense.